src/java/install.sh

```diff
@@ -111,13 +111,13 @@ sdk_install() {
     elif echo "${requested_version}" | grep -oE "${full_version_check}" > /dev/null 2>&1; then
         echo "${requested_version}"
     else
-        local regex="${prefix}\\K[0-9]+\\.[0-9]+\\.[0-9]+${suffix}"
+        local regex="${prefix}\\K[0-9]+\\.?[0-9]*\\.?[0-9]*${suffix}"
```

Comment:
Can we update the minor version of the Feature, thanks! 

https://github.com/devcontainers/features/blob/d53f4f1c1d9cc34e537d32188d9ca5341467918a/src/java/devcontainer-feature.json#L3

Reply:
Bumped patch version!